patches/node/fix_tolocalstring_unicode_mismatch.patch

```diff
@@ -13,15 +13,14 @@ This patch can be removed when https://github.com/nodejs/node/pull/45068
 is backported to Node.js v18.
 
 diff --git a/test/parallel/test-intl.js b/test/parallel/test-intl.js
-index 7d1742f2c7d1c6e335d3ccfc81ce0834c1de9c19..37d9749c684c4fe12f9c06d8bee9a16ea15bb708 100644
+index 7d1742f2c7d1c6e335d3ccfc81ce0834c1de9c19..df391bd4de3db9a41f3274ca678c9034be9255fe 100644
 --- a/test/parallel/test-intl.js
 +++ b/test/parallel/test-intl.js
-@@ -97,7 +97,8 @@ if (!common.hasIntl) {
+@@ -97,7 +97,7 @@ if (!common.hasIntl) {
    // Test format
    {
      const localeString = date0.toLocaleString(['en'], optsGMT);
 -    assert.strictEqual(localeString, '1/1/1970, 12:00:00 AM');
-+    console.log(localeString)
 +    assert.strictEqual(localeString, '1/1/1970, 12:00:00 AM');
```

Comment:
ahh... one of these has U+202F between "12:00:00" and "AM", the other has U+0020.

Reply:
and this is written in the patch description :+1: